muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/toolbar/labels.tsx

```diff
@@ -92,7 +92,10 @@ class GuiderToolbarLabels extends React.Component<
         <Dropdown
           modifier="guider-labels"
           items={[
-            <div key="update-label" className="form-element">
+            <div
+              key="update-label"
+              className="form-element form-element--new-flag"
```

Comment:
We should name this flag the same as communicator labels as we already use update-label as key attribute's value.

Then there really is no need to add multiple scss rules when we call them labels instead of separately flags. Everything else seems to be named as label internally in the code already.